settings.gradle.kts

```diff
@@ -13,6 +15,37 @@ dependencyResolutionManagement {
         google()
         gradlePluginPortal()
         mavenCentral()
+        maven(
+            "https://maven.pkg.github.com/govuk-one-login/mobile-android-networking",
+            setupGithubCredentials()
+        )
+    }
+}
+
+fun setupGithubCredentials(): MavenArtifactRepository.() -> Unit =
+    {
+        val (credUser, credToken) = fetchGithubCredentials()
+        credentials {
+            username = credUser
+            password = credToken
+        }
+    }
+
+fun fetchGithubCredentials(): Pair<String, String> {
+    val gprUser = providers.gradleProperty("gpr.user")
+    val gprToken = providers.gradleProperty("gpr.token")
+
+    return try {
+        gprUser.get() to gprToken.get()
+    } catch (exception: MissingValueException) {
+        logger.warn(
+            "Could not find 'Github Package Registry' properties. Refer to the proceeding " +
+                    "location for instructions:\n\n" +
+                    "${rootDir.path}/docs/developerSetup/github-authentication.md\n",
+            exception
+        )
+
+        System.getenv("USERNAME") to System.getenv("TOKEN")
```

Comment:
Do we still need all of this since not using the networking package here?

Reply:
No, good spot. I thought I'd got rid of all the build stuff